apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -49,6 +50,31 @@ export function HeaderNav({}: Props) {
     }, [currentUser, currentOrganization]);
   }
 
+  if (LOGROCKET_ID) {
+    useEffect(() => {
+      if (currentUser && currentOrganization) {
+        let logrocketTraits;
+
+        if (currentUser?.email !== undefined) {
+          logrocketTraits = {
+            name: currentUser?.firstName + ' ' + currentUser?.lastName,
+            organizationId: currentOrganization._id,
+            organization: currentOrganization.name,
+            email: currentUser?.email ? currentUser?.email : ' ',
+          };
+        } else {
+          logrocketTraits = {
+            name: currentUser?.firstName + ' ' + currentUser?.lastName,
+            organizationId: currentOrganization._id,
+            organization: currentOrganization.name,
+          };
+        }
+
+        LogRocket.identify(currentUser?._id, logrocketTraits);
+      }
+    }, [currentUser, currentOrganization]);
+  }
```

Comment:
please move the `if (LOGROCKET_ID)` to be inside the `useEffect`

```suggestion
    useEffect(() => {
      if (!LOGROCKET_ID) return;
      
      if (currentUser && currentOrganization) {
        let logrocketTraits;

        if (currentUser?.email !== undefined) {
          logrocketTraits = {
            name: currentUser?.firstName + ' ' + currentUser?.lastName,
            organizationId: currentOrganization._id,
            organization: currentOrganization.name,
            email: currentUser?.email ? currentUser?.email : ' ',
          };
        } else {
          logrocketTraits = {
            name: currentUser?.firstName + ' ' + currentUser?.lastName,
            organizationId: currentOrganization._id,
            organization: currentOrganization.name,
          };
        }

        LogRocket.identify(currentUser?._id, logrocketTraits);
      }
    }, [currentUser, currentOrganization]);
```